acceptance/tests/validate_vendored_ruby.rb

```diff
@@ -10,7 +10,7 @@ def package_installer(agent)
   # need to manually set up the package installer for these platforms.
   case agent['platform']
   when /aix/
-    lambda { |package| on(agent, "rpm -Uvh #{package}") }
+    lambda { |package| on(agent, "curl -O #{package} && rpm -Uvh `basename #{package}`") }
```

Comment:
Should we try to delete an existing package before curling, for example when re-running the test?